recipes/grass/meta.yaml

```diff
@@ -0,0 +1,101 @@
+{% set version = "7.4.2" %}
+
+package:
+  name: grass
+  version: {{ version }}
+
+source:
+  url: https://grass.osgeo.org/grass74/source/grass-{{ version }}.tar.gz
+  sha256: 18eb19bc0aa4cd7be3f30f79ac83f9d0a29c63657f4c1b05bf4c5d5d57a8f46d
+  patches:
+    - platform.make.in.patch  # [osx]
+    - platform.make.in-linux.patch  # [linux]
+    - loader.py.patch
+    - rules.make.patch  # [osx]
+    - rules.make-linux.patch  # [linux]
+    - aclocal.m4.patch
+    - configure.patch
+    - install.make.patch
+    - module.make.patch  # [osx]
+    - shlib.make.patch  # [osx]
+
+build:
+  number: 0
+  skip: True  # [not osx]
```

Comment:
Did you mean to skip osx?

Reply:
I'm pretty sure this skips everything *except* osx, right?

Reply:
Right. I got confused after seeing the build statuses passing for everything but osx. I should have reread your first comment. Glad you created this recipe. Having GRASS available through conda-forge will be nice.